sisyphus/localengine.py

```diff
@@ -282,23 +307,18 @@ def try_to_recover_task(self, task, task_id):
             rqmt = d['requested_resources']
             logpath = os.path.relpath(task.path(gs.JOB_LOG_ENGINE))
             call_with_id = task.get_worker_call(task_id)
-            call_with_id += ['--redirect_output']
             name = task.task_name()
             task_name = task.name()
             task_instance = TaskQueueInstance(call_with_id, logpath, rqmt, name, task_name, task_id)
 
-            if call_with_id != process.cmdline()[1:]:
+            if call_with_id[1:] != process.cmdline()[1:]:
                 logging.debug('Job changed, ignore this job: %i %s %s' % (pid, process.cmdline(), task_instance.call))
```

Comment:
This message is now a little misleading, it should be something like "Job change, recovering it anyway". I think we should also promote this from a debug to a warning message.

Reply:
I left the more technical message as debug and added a more sensible warning containing the name of the job in question .